app/app.py

```diff
@@ -33,27 +36,27 @@ def _get_fragment_metadata(fragment_id: str, mh_client: MediahavenService) -> Di
 
     Arguments:
         fragment_id {str} -- Fragment ID for which the information is fetched.
-        mh_client {MediahavenService} -- The MH client.
+        mh_client {Mediahaven} -- The MH client.
 
     Returns:
         Dict[str, str] -- Dictionary containing the retrieved metadata.
     """
 
     try:
-        fragment = mh_client.get_fragment(fragment_id)
-    except MediaObjectNotFoundException as error:
+        fragment = mh_client.records.get(fragment_id)
```

Comment:
```suggestion
        fragment = mh_client.records.get(fragment_id).single_result
```
This way you don't need to repeat it later on, lines 55-58.